src/actions/auth.js

```diff
@@ -23,15 +23,19 @@ exports.registrationFailure = (errMessage) => ({
     payload: errMessage
 });
 
+exports.loginFailure = (errMessage) => ({
```

Comment:
This is redundant because Strange Auth already dispatches this action: 
  * https://github.com/BigRoomStudios/strange-auth/blob/master/src/actions.js#L24-L31
  * https://github.com/BigRoomStudios/strange-auth/blob/master/src/types.js#L8